gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -58,6 +58,8 @@
 #define VENDOR_PAGES_GROUP "pages"
 #define VENDOR_PAGES_SKIP_KEY "skip"
 
+#define PAGES_DEFAULT_SORT_WEIGHT 100
```

Comment:
This `#define` is now unused and should be removed.

Reply:
Sorry about that.